src/main/kotlin/no/nav/klage/oppgave/egenansatt/EgenAnsattService.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.klage.oppgave.egenansatt
+
+import org.springframework.stereotype.Service
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentMap
+
+@Service
+class EgenAnsattService {
+
+    private val egenAnsattMap: ConcurrentMap<String, EgenAnsatt> = ConcurrentHashMap()
+
+    fun erEgenAnsatt(fodselsnr: String): Boolean =
+        egenAnsattMap[fodselsnr]?.erGyldig() ?: false
```

Comment:
Ikke 100 % relevant kanskje, siden det ikke er i en condition, men er interessant: https://kotlinlang.org/docs/reference/coding-conventions.html#using-nullable-boolean-values-in-conditions

Reply:
Ja, det er jo ikke så rart. Hvis man bare skriver `if ( Boolean-value )` og den kan være null, hva blir da null evaluert til? Så det er et godt råd, men ikke gjeldende her.